app/src/main/java/org/mozilla/fenix/home/recentsyncedtabs/RecentSyncedTabFeature.kt

```diff
@@ -129,6 +148,12 @@ class RecentSyncedTabFeature(
     }
 }
 
+/**
+ * The number of days to search history for a preview image URL to display for a synced
+ * tab.
+ */
+private const val DAYS_HISTORY_FOR_PREVIEW_IMAGE = 3L
```

Comment:
Should this just be inside the `companion object` of `RecentSyncedTabFeature`? No strong opinions on this, but it might be something worth doing to make it testable.

Reply:
I hadn't thought of it from a testability standpoint. A companion object would probably make the most sense in that case. I generally kinda dislike the pattern of instantiating a whole object just for a const val, [this SO post](https://stackoverflow.com/questions/44038721/constants-in-kotlin-whats-a-recommended-way-to-create-them) covers why a bit. A top-level internal val could work if we had modules in Fenix, but I think namespacing it with a companion makes more sense since we don't.

Reply:
Hmm.. interesting. I don't know if we really considered the pattern about how we define constants. My initial reaction is simply based on what I have seen in the project, and looking at what we did in the `BookmarksUseCase`. I won't be the best person to weigh on how we should ideally proceed with this, but might always be something to bring up in a discussion if you think it's worthwhile to establish some sort of guidelines.